var/spack/repos/builtin/packages/python/package.py

```diff
@@ -873,8 +873,7 @@ def command(self):
                 if os.path.exists(path):
                     return Executable(path)
 
-            msg = "Unable to locate {0} command in {1}"
-            raise RuntimeError(msg.format(self.name, self.prefix.bin))
+        raise RuntimeError(f"Unable to locate {self.name} command in {self.prefix.bin}")
```

Comment:
This error should probably be updated to reflect idiomatic location on Windows

Reply:
Updated